tcp_server.js

```diff
@@ -0,0 +1,22 @@
+var net = require('net');
+var fs = require('fs');
+var chai = require('chai'), chaiHTTP = require('chai-http');
+
+chai.use(chaiHTTP);
```

Comment:
Chai and chai-http are used for testing. (And chai-http isn't very useful when testing a TCP server.)